docs/playbooks/concepts/playbook/examples.md

```diff
@@ -0,0 +1,26 @@
+# Examples
+
+## 1. Scaling EC2 instance
+
+```yaml
+description: Scale deployment
+configs:
+  - type: Kubernetes::Deployment
+    tags:
+      namespace: default
+      cluster: local-kind-cluster
+parameters:
+  - name: replicas
+    label: The new desired number of replicas.
+approval:
+  type: any
+  approvers:
+    people:
+      - d87243c9-3183-4ab9-9df9-c77c8278df11
```

Comment:
These should be names or emails - not ids

Reply:
@moshloop will make this email only. name is not unique.